### PR TITLE
0308 erin batch

### PR DESCRIPTION
**Pull request by mumian**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: quickstarts/microsoft.batch/batchaccount-with-storage/main.bicep [hunk 1]
```diff
@@ -0,0 +1,49 @@
+@description('Batch Account Name')
+param batchAccountName string = '${toLower(uniqueString(resourceGroup().id))}batch'
+
+@description('Storage Account type')
+@allowed([
```
Comment (bmoore-msft) [inline quickstarts/microsoft.batch/batchaccount-with-storage/main.bicep:5]: I don't think batch supports all of these account types

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Comment by JnHs** [comment]
> @JnHs - check this PR for updates that may be needed to documentation that references this sample. [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

I am no longer the owner of this document, it's now @lauradolan (who I can't seem to directly tag for some reason)

**Comment by azure-quickstarts** [comment]
@JnHs - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.batch/batchaccount-with-storage/main.bicep (hunk 1 above)
Comment (mumian) [reply quickstarts/microsoft.batch/batchaccount-with-storage/main.bicep:5]: Preetha Bannu Kannappan (Azure Batch PM) will find out the supported SKUs.  Until then, revert the SKU list.

**Pull request merged**